conf/jvm17-server.options

```diff
@@ -92,12 +92,15 @@
 --add-opens java.base/java.nio=ALL-UNNAMED
 --add-opens jdk.compiler/com.sun.tools.javac=ALL-UNNAMED
 
+# required for org.apache.cassandra.Util.getSupportedMTimeGranularity
+--add-opens java.base/java.nio.file.attribute=ALL-UNNAMED
+
 ### GC logging options -- uncomment to enable
 
 # Java 11 (and newer) GC logging options:
 # See description of https://bugs.openjdk.java.net/browse/JDK-8046148 for details about the syntax
 # The following is the equivalent to -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=10M
--Xlog:gc=info,heap*=debug,age*=debug,safepoint=info,promotion*=debug:file=/var/log/cassandra/gc.log:time,uptime,pid,tid,level:filecount=10,filesize=10485760
+# -Xlog:gc=info,heap*=debug,age*=debug,safepoint=info,promotion*=debug:file=/var/log/cassandra/gc.log:time,uptime,pid,tid,level:filecount=10,filesize=10485760
```

Comment:
why is this being commented out?

Reply:
It is cherry-pick from JDK11 fix - [CC5 fixes dtests by not hardening gc.log location - same as](https://github.com/datastax/cassandra/pull/1408/commits/250eaa0a63864d3ccd41f41e3047539440ee3887) https://github.com/datastax/cassandra/commit/61589a25d2dc8c18949dd6962c9c80fd3f5bc1f1 [for JDK11](https://github.com/datastax/cassandra/pull/1408/commits/250eaa0a63864d3ccd41f41e3047539440ee3887)

I think otherwise it breaks CI. Also, on Mac it won't work out of the box so I welcomed the change. Maybe @driftx can say more here.

Reply:
It's already [here](https://github.com/datastax/cassandra/blob/main/conf/cassandra-env.sh#L100) which respects CASSANDRA_LOG_DIR so non-root users can run it.

Reply:
yeah, you should do the same for `main`